pages/main/main.css

```diff
@@ -92,10 +105,189 @@ h3 {
 
 .about-add-text {
 	font-family: 'Arial';
-	font-style: normal;
+
 	font-weight: 400;
 	font-size: 15px;
 	line-height: 160%;
 	color: var(--color-dark-xl);
 	margin-bottom: 6px;
+}
+
+.pets-slider{
+  display: flex;
+}
+
+.help-container, .our-friends{
+  background-color: #F6F6F6;
+  padding: 80px 0 100px;
+}
+
+.donation-wrapper{
+  padding: 80px 0 100px;
+}
+
+.arrow-right{
```

Comment:
Arrows should be in the circles, for now, they look more like to are in the ovals.

Your Design | Figma Design
--------------- | ------------
![arrow](https://user-images.githubusercontent.com/79588769/166154479-0324de03-2498-4328-a438-950fb6609485.png) | ![arrowF](https://user-images.githubusercontent.com/79588769/166154483-6e6daaf2-bbfe-4593-a941-81f1fcec9598.png)